.github/validate_customizations.py

```diff
@@ -0,0 +1,166 @@
+import json
+import pathlib
+import sys
+
+
+def scrub(txt: str) -> str:
+	"""Returns sluggified string. e.g. `Sales Order` becomes `sales_order`."""
+	return txt.replace(" ", "_").replace("-", "_").lower()
+
+
+def unscrub(txt: str) -> str:
+	"""Returns titlified string. e.g. `sales_order` becomes `Sales Order`."""
+	return txt.replace("_", " ").replace("-", " ").title()
+
+
+def get_customized_doctypes():
+	apps_dir = pathlib.Path(__file__).resolve().parent.parent.parent
+	apps_order = pathlib.Path(__file__).resolve().parent.parent.parent.parent / "sites" / "apps.txt"
+	apps_order = apps_order.read_text().split("\n")

```

Comment:
I'm getting an error here.
The path is  `.../bench_folder/apps/sites/apps.txt` instead of `../bench_folder/sites/apps.txt`. I'm running it from `bench_folder` in a bench console.